screens/Registration/AuthenticationScreen/AuthenticationScreen.tsx

```diff
@@ -69,11 +70,17 @@ const AuthenticationScreen: React.FC = () => {
   }
 
   return (
-    <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
+    <TouchableWithoutFeedback 
+    style={[
+      styles.container, 
+      {height: Dimensions.get('window').height + insets.bottom + insets.top}
+    ]}
+      onPress={() => Keyboard.dismiss()}>
       <View
         style={[
           styles.view,
           { paddingBottom: insets.bottom, paddingTop: insets.top },
+          {height: Dimensions.get('window').height + insets.bottom + insets.top}
```

Comment:
Why did you specify height, was `flex: 1` or `height: "100%"` not enough for both styles?